angularSTEP/src/app/auto-complete/auto-complete.component.ts

```diff
@@ -12,7 +12,7 @@ export class AutoCompleteComponent implements OnInit {
 
   myControl = new FormControl();
   options: string[] = ['One', 'Two', 'Three'];
-  filteredOptions: Observable<string[]>;
+  filteredOptions!: Observable<string[]>;
```

Comment:
The exclamation point is needed because the Google typescript checker throws an error if a value is declared but not initiated within the constructor - not trying to comment for Dom, however, I am adding my review and thought I would put this in.

Reply:
yeah Without "!" it would throw errors